### PR TITLE
fix some bug about gulp, now it works on gulp.

### PR DESCRIPTION
**Pull request by kevinwon1985**

### CODE REVIEW

File: templates/common/root/_gulpfile.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // Generated on <%= (new Date).toISOString().split('T')[0] %> using <%= pkg.name %> <%= pkg.version %>
+// Generated on 2015-11-11 using generator-angular 0.14.0
```
Comment (eddiemonge) [inline templates/common/root/_gulpfile.js]: this should not be here

**Comment by eddiemonge** [comment]
What is this change for? Can you provide a problem it solves and how it solves it? Also the commit message doesn't follow the contributing guidelines

**Comment by kevinwon1985** [comment]
@eddiemonge fix the bug in this issue #1199 , I'm sorry for I had not follow  the contributing guidelines.
Becouse my english is poor.

**Comment by eddiemonge** [comment]
closing due to conflicts and other fixes needed

**Pull request closed without merging**